docs/en/writing-running-appium/caps.md

```diff
@@ -64,6 +64,8 @@ These Capabilities span multiple drivers.
 |----|-----------|-------|
 |`settings[settingsKey]`| Update [Appium Settings](https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/settings.md) on session creation. |e.g., `'settings[mjpegScalingFactor]': 10`, `'settings[shouldUseCompactResponses]': true`|
 
+
+
```

Comment:
and here are redundant new lines

Reply:
@KazuCocoa thanks for so quickly feedback, i  have update the caps document. 😋

Reply:
Could you remove L67 and 68's new lines, too?

Reply:
updated, thanks